app/helpers/games_helper.rb

```diff
@@ -1,6 +1,9 @@
 module GamesHelper
   def render_piece(x, y)
-    piece = @game.pieces.find_by(x_position: x, y_position: y)
-    return "#{piece.type} #{piece.color} " if piece
+    if piece = @game.pieces.find_by(x_position: x, y_position: y)
+     return link_to "#{piece.type} #{piece.color}", piece_path
+    else
+     return "Move Too"
+   end 
```

Comment:
For highlighting the piece. This is kind of hacky, but you could do something like...

(this code is incomplete, but hopefully you get the idea)
```Ruby
if @piece.present? #we know this is set only in the pieces controller
  if @piece.x_coordinate == x && @piece.y_coordinate == y
    # return a text that is highlighted to show this piece is selected
   end
end
```